phone/src/apps/twitter/components/TwitterApp.tsx

```diff
@@ -35,6 +36,17 @@ const useStyles = makeStyles(() => ({
 
 const TWEETS_REFRESH_RATE = 15000; // TODO move this to twitter config
 
+InjectDebugData([
```

Comment:
We don't really need this, but whatever

Reply:
Include or not, I moved all the debug events from Phone.tsx